docs/platforms/javascript/common/tracing/distributed-tracing/index.mdx

```diff
@@ -27,6 +27,16 @@ If you run any JavaScript applications in your distributed system, make sure tha
 
 </Alert>
 
+<PlatformSection supported={["javascript.nextjs"]}>
+
+<Alert>
+
+Pageload to SSR span linking is not supported on the App Router with Next.js versions up to and including 14.
```

Comment:
l: let's be clear on terminology. [Span linking](https://develop.sentry.dev/sdk/telemetry/traces/span-links/) is different to tracing and we should avoid confusing users :)

```suggestion
Pageload to SSR tracing is currently not supported on the App Router with Next.js versions up to and including 14.
```